app/herdbook.py

```diff
@@ -474,6 +480,250 @@ def get_mean_kinship(g_id):
     return {}
 
 
+@APP.route("/api/certificates/update/<i_number>", methods=["POST"])
+@login_required
+def update_certificate(i_number):
+    """
+    Returns an updated pdf of the individual given by `i_number`.
+    """
+    user_id = session.get("user_id", None)
+    ind = da.get_individual(i_number, user_id)
+
+    if ind is None:
+        return jsonify({"response": "Individual not found"}), 404
+
+    certificate_exists = ind.get("certificate", None)
+    if certificate_exists and check_certificate_s3(object_name=ind["number"]):
```

Comment:
```suggestion
    if certificate_exists and check_certificate_s3(object_name="certificate/" + ind["number"]):
```

Alternatively `ind["number"] + "/certificate"`, feels like a good idea to keep the generic identifier free if we possibly  want to add other storage objects.